### DISCUSSION TITLE
Microsoft Visual C++ 14.0 or greater is required

### DISCUSSION THREAD

**Discussion by WSrujan** [ThioJoe/YT-Spammer-Purge#737] [General]
I can't get require package to install on python.
It says to install Microsoft Visual C++ 14.0 or greater is required. so i did its 3GB big where 2.7GB is bloatware like (debugger,compiler,rest of build tools)

Is there way fix this error without bloatware i know to will just say to install redistribute visual C++ but it does not work it needs specific file to work and its in full of bloatware>

I am asking since i don't want rest of stuff ( i do run different codes of python and other languages so .exe  not a option for me

**Comment by Masamune3210** [comment]
It doesnt want the full suite, just the Redistributable

**Reply by WSrujan** [reply]
I did installed it
Latest one
